gradle-language-server/src/main/java/com/microsoft/gradle/resolver/GradleLibraryResolver.java

```diff
@@ -209,6 +217,8 @@ private File findLibFolder(DistInfo info) {
             // libFolder matches
             // .gradle/wrapper/dists/${gradleDist}/internal-string/${gradleVersion}/lib
             if (Utils.isValidFolder(libFolder)) {
+              this.gradleLibraryFolder = libFolder;
+              this.gradleFilesManager.setGradleLibraries(this);
```

Comment:
why not to pass all gradle libraries to setGradleLibraries directly?

Reply:
I'll change it.